.gitlab-ci.yml

```diff
@@ -297,3 +299,25 @@ ansible-release:
   script:
     - export COLLECTION_VERSION=$(cat ./deployments/ansible/galaxy.yml | yq .version -r)
     - ansible-galaxy collection publish ./dist/signalfx-splunk_otel_collector-${COLLECTION_VERSION}.tar.gz --token=${ANSIBLE_GALAXY_TOKEN} 
+
+puppet-release:
+  image: 'docker-hub.repo.splunkdev.net/ruby:2.6-buster'
+  stage: release
+  only:
+    variables:
+      - $CI_COMMIT_TAG =~ /^puppet-v[0-9]+\.[0-9]+\.[0-9]+.*/
+  cache:
+    key: "puppet-bundler-cache"
+    paths:
+      - .cache/bundler
```

Comment:
Do we really need to cache the bundler stuff assuming puppet release happens not that often? Wouldn't be better to use an up-to-date bundler installation instead?

Reply:
Removed bundler cache